### DISCUSSION TITLE
[5.x]: Show sidebar toggle for entry preview panels should be persistent/have a default state setting

### DISCUSSION THREAD

**Discussion by turnstylerj** [craftcms/cms#14798] [Ideas]
## Description

When using new Matrix fields for a page-builder type setup, and the Title/Slug/Status fields disabled for Entry Types used within the Matrix field, the sidebar takes up a lot of space displaying effectively useless information to the author, especially when we get into 2nd or 3rd-level slide-out views.

![2024-04-11 at 16 55 39](https://github.com/craftcms/cms/assets/42223501/e53d0a99-6d24-4954-be92-6e4a99c7c0a3)

These can be hidden with the "Show sidebar" toggle, but the selection is not persistent so every time a Matrix entry is created/opened, the author needs to manually toggle the sidebar off again if they want to hide it.

### Option 1 (potentially simpler)

"Show sidebar" option is persistent and maintains its state for a given entry when the author selects it.

![2024-04-11 at 16 41 09](https://github.com/craftcms/cms/assets/42223501/bd7b79ce-ca84-4c58-9327-894caf71c158)

### Option 2 (better)

Sidebar display is configurable to be collapsed by default in an Entry Type's settings.

![2024-04-11 at 16 53 18](https://github.com/craftcms/cms/assets/42223501/6beaed92-b5a4-4cdb-8c70-2c5c9bb18a48)

**Comment by ryanleichty** [comment]
This same issue is mentioned here as well: https://github.com/craftcms/cms/discussions/14991
